packages/turbo-gen/src/generators/custom.ts

```diff
@@ -32,9 +34,25 @@ export async function generate({
         throw err;
       }
 
+      // make it obvious that we're done creating a generator, and now we're running it
+      console.log();
+      logger.info(`Example generator config successfully created!`);
+      logger.info(`Loading generator config...`);
+      console.log();
+
       // fetch generators again, and continue to selection prompt
       generators = getCustomGenerators({ project, configPath: opts.config });
+
+      // something went wrong and we weren't able to find our new demo generator
+      if (!generators.length) {
+        logger.error(`Error loading generator.`);
+        return;
+      }
     } else {
+      console.log();
+      logger.dimmed(
+        "Learn more about generators: https://turbo.hotwire.dev/reference/generators"
```

Comment:
lol what is this URL? did copilot generate this for you!?

Reply:
Yea copilot was convinced this is what the URL should be for all the docs. I've been fighting this for the entire project and these two must have slipped through 😂 

I am on the chat beta, so maybe there is some wonkiness there. 

Thanks for the catch!